src/molecule/provisioner/ansible_playbook.py

```diff
@@ -103,12 +105,16 @@ def execute(self):
             LOG.warning("Skipping, %s action has no playbook." % self._config.action)
             return
 
-        self._config.driver.sanity_checks()
-        result = util.run_command(self._ansible_command, debug=self._config.debug)
+        with warnings.catch_warnings(record=True) as warns:
+            warnings.filterwarnings("default", category=MoleculeRuntimeWarning)
+            self._config.driver.sanity_checks()
```

Comment:
I think we should only catch warnings coming from this invocation.

Reply:
That is what is happening with record, when the context is destroyed the filter is also reset. At least that is what docs are saying.

Reply:
I meant that we shouldn't suppress those warnings in `run_command()`